.github/actions/generate-resources/main.js

```diff
@@ -78,19 +78,11 @@ const init = async () => {
 
   // Write README.md file
   fs.writeFileSync(
-    './README.md',
+    './RESOURCES.md',
     `${README_RESOURCE_HEADER}${README_RESOURCE_BODY}`,
   );
 };
 
-
-/**
- * On the condition of flag node main.js --run
- */
-if (process.argv && process.argv.includes('--run')) {  
-  init();
-}
-
```

Comment:
Remove this because in the `action.yml` doesn't support flags like `--run`, which I needed to make sure it wasn't automatically run with jest tests.